scenario_player/environment/goerli-latest.json

```diff
@@ -0,0 +1,15 @@
+{
+    "environment_type": "development",
+    "matrix_server_list": "https://raw.githubusercontent.com/raiden-network/raiden-service-bundle/goerli-latest-servers/known_servers/known_servers-goerli-latest.json",
+    "pfs_with_fee": "https://pfs.goerli-latest1.env.raiden.network",
+    "eth_rpc_endpoints": [
+        "http://geth.goerli.ethnodes.brainbot.com:8545"
+    ],
+    "transfer_token": "0xC563388e2e2fdD422166eD5E76971D11eD37A466",
+    "pfs_fee": 100,
+    "ms_reward_with_margin": 5500000000000000000,
+    "settlement_timeout_min": 40,
```

Comment:
Just FYI, the 0.50 `deployment_goerli.json` sets the `settle_timeout` to 15k seconds (~4h). This value here would need to match that, but either way, 4h is quite too long for settle_timeout on scenarios. We may want to redeploy those with reasonable testnet timeouts of 180s (e.g. as done in Arbitrum Rinkeby contracts). No need to worry now anyway